android/src/com/mapswithme/maps/MwmActivity.java

```diff
@@ -841,6 +843,7 @@ protected void onSaveInstanceState(@NonNull Bundle outState)
     RoutingController.get().onSaveState();
     outState.putBoolean(EXTRA_LOCATION_DIALOG_IS_ANNOYING, LocationHelper.INSTANCE.isLocationErrorDialogAnnoying());
     outState.putInt(EXTRA_CURRENT_LAYOUT_MODE, mCurrentLayoutMode.ordinal());
+    outState.putBoolean(EXTRA_IS_FULLSCREEN, mIsFullscreen);
```

Comment:
Why do you think that saving the full-screen mode between app restarts is a good idea? It may confuse the user when he/she opens the app after several days.

Reply:
My bad, you're right )